src/node/rpc/member_frontend.h

```diff
@@ -1967,7 +2008,7 @@ namespace ccf
         }
 
         if (!set_jwt_public_signing_keys(
-              ctx.tx, INVALID_ID, parsed.issuer, issuer_metadata, parsed.jwks))
+              ctx.tx, "", parsed.issuer, issuer_metadata, parsed.jwks))
```

Comment:
Nit: Would prefer if `""` was `INVALID_PROPOSAL_ID`, set to something like `INVALID_PROPOSAL_ID = "INVALID"` in `proposals.h`, so we can catch if this ever leaks somewhere unexpected.

Reply:
Ok